scopesim_templates/metis/pinhole_mask.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""Pinhole mask for METIS."""
+
+from itertools import product
+
+import numpy as np
+from astropy import units as u
+
+from ..micado.pinhole_masks import pinhole_mask as micado_mask
+
+
+def pinhole_mask(
+        nx=5,
+        ny=28,
+        dx=0.45,
+        dy=0.27945,
+        waves=np.arange(3, 6, .001)*u.um,
+        **kwargs
+):
+    """
+    Create pinhole mask source for METIS, simulated as a grid of point sources.
+
+    This currently uses the (flexible)  MICADO pinhole mask internally.
+
+    Parameters
+    ----------
+    nx : int, optional
+        Number of points along x axis. The default is 5.
+    ny : int, optional
+        Number of points along y axis. The default is 28.
+    dx : float, optional
+        (Half) extension in x direction. The default is 0.45.
+    dy : float, optional
+        (Half) extension in y direction. The default is 0.27945.
+    waves : array-like, optional
+        Wavelength axis. The default is np.arange(3, 6, .001)*u.um.
+    **kwargs
+        Any other kwargs passed to micado.pinhole_mask.
```

Comment:
Perhaps write these out explicitly? Because I was wondering about what spectrum would be used, and had to look this up in the micado function.